includes/discovery/sensors/temperature/axiscam.inc.php

```diff
@@ -2,6 +2,39 @@
 
 echo 'AXIS Temperatures ';
 
+// Set Temp Limits based on Device Description
+switch (true) {
+    case stristr($device['sysDescr'], 'P1425-LE'):
+        $high_limit = '50';
+        $high_warn_limit = '40';
+        $low_warn_limit = '-20';
+        $low_limit = '-30';
+        break;
+    case stristr($device['sysDescr'], 'P1435-LE'):
+        $high_limit = '60';
+        $high_warn_limit = '50';
+        $low_warn_limit = '-20';
+        $low_limit = '-30';
+        break;
+    case stristr($device['sysDescr'], 'P1455-LE'):
+        $high_limit = '60';
+        $high_warn_limit = '50';
+        $low_warn_limit = '-30';
+        $low_limit = '-40';
+        break;
+    case stristr($device['sysDescr'], 'P5676-LE'):
+        $high_limit = '50';
+        $high_warn_limit = '40';
+        $low_warn_limit = '-20';
+        $low_limit = '-30';
+        break;
+    default:
+        $high_limit = null;
```

Comment:
should the default be the old values?

You can remove the quotes around all the numbers and just let them be integers.

Reply:
We could also do that, i removed them because it caused all devices to alert (since most axis cameras are also outdoor cams that support way higher temperatures than just 35 °).